### ISSUE TITLE
Annotation: improve volume layer handling

### ISSUE THREAD

**Issue by jstriebel** [scalableminds/webknossos-libs#623]
* [ ] Once https://github.com/scalableminds/webknossos/issues/6127, #688 and https://github.com/scalableminds/webknossos/issues/6193 are in place, we should provide methods to retrieve the (possibly merged) volume layers as a dataset / layer.
* [ ] Add shortcut to create an annotation with a fallback layer directly from a `Layer`.

---------

**Old issue description for reference**

Handling volume layers in annotations should be improved. Some points, from high to lower priority
* allow to merge volume layer into the respective fallback layer
* add shortcut to create an annotation with a fallback layer directly from a `Layer`
* Allow to merge multiple volume annotations into a single one
* Consider a dataset-less layer class to represent and interact with volume annotations directly, but the tmp copy seems fine for this, too.

PS: We should also add tests for annotation download & upload, e.g. a roundtrip test.

**Comment by jstriebel** [comment]
According to the conversation in #655, we should prioritize merging volume annotation data with the respective fallback layer.

@erjel, in your use-case, do you merge the annotation data into the original fallback layer, or do you produce another separate layer with the merged data? It would be interesting to see what's your default use-case here. Thanks a lot for the insights already!

**Comment by erjel** [comment]
The final goal is to save the annotation into something which 
- represents 1:1 the pixel values in the web UI visualization
- can be converted into numpy arrays (i.e. a local wkw dataset).
- is portable and can be moved around without server access (i.e. offline compute clusters).

That was the reason why I was surprised by the "diff" annotation download. This works in the case that you have file system access to the storage server and can mount it in a compute instance. Yet, the majority would expect download button (or API call) which directly produces a local wkw dataset which covers the previously mentioned bullet points.

**Comment by jstriebel** [comment]
blocked by https://github.com/scalableminds/webknossos/issues/6270
